packages/xod-client/src/debugger/selectors.js

```diff
@@ -186,7 +186,16 @@ export const getInteractiveNodeValuesForCurrentPatch = createSelector(
               sheets[lastSheetIndex].length
             }`;
           }),
-          filterOutValuesForCurrentPatch(activeIndex, nodeIdPath)
+          filterOutValuesForCurrentPatch(activeIndex, nodeIdPath),
+          // `tsv-log`, which is actually stores the data, encapsulated
+          // inside `table-log`, which label should be replaced with
+          // this value. So we have to cut the latest nodeId part from
+          // keys of table log values map
```

Comment:
It could be done with something like `R.replace(/(~[^~]+)$/, '')`
But your version is probably readable 😀